src/utils/zcl_abapgit_convert.clas.abap

```diff
@@ -98,11 +92,7 @@ CLASS ZCL_ABAPGIT_CONVERT IMPLEMENTATION.
 
   METHOD alpha_output.
 
-    CALL FUNCTION 'CONVERSION_EXIT_ALPHA_OUTPUT'
-      EXPORTING
-        input  = iv_val
-      IMPORTING
-        output = rv_str.
+    rv_str = |{ iv_val ALPHA = OUT }|.
```

Comment:
Hmm, isn't it a 7.40 feature ? `ALPHA = OUT` ? (cannot check immediately but I'm 99% sure it is)

Reply:
true, https://github.com/abapGit/abapGit/issues/4178 opened